### ISSUE TITLE
Question about Masked Scene Contrast evaluation

### ISSUE THREAD

**Issue by ywyue** [Pointcept/Pointcept#358]
Hi @Gofinge, thanks for your great work! I have some questions regarding the evaluation of Masked Scene Contrast.

- If I understand correctly, in tab. 2(a) first row, MSC represents the results of pretraining on ScanNet + ARKitScenes and finetuning on ScanNet for semantic segmentation. SC denotes train from scratch. I wonder does SC denotes train from scratch **only on ScanNet** or **on ScanNet + ARKitScenes**?
- Tab. 1 shows the results of the model pretrained only on ScanNet. Did you compare this model with model training from scratch on ScanNet?

Thank you in advance!

**Comment by Gofinge** [comment]
> If I understand correctly, in tab. 2(a) first row, MSC represents the results of pretraining on ScanNet + ARKitScenes and finetuning on ScanNet for semantic segmentation. SC denotes train from scratch. I wonder does SC denotes train from scratch only on ScanNet?

Yes, sc only on ScanNet. No semantic label for ArkitScene

> Tab. 1 shows the results of the model pretrained only on ScanNet. Did you compare this model with model training from scratch on ScanNet?

Maybe needs our reader to manually compare with the SC result on ScanNet, it is the same as the one in the main result table.
